awx/ui_next/package.json

```diff
@@ -32,6 +32,7 @@
   },
   "scripts": {
     "start": "PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts start",
+    "start-instrumented": "DEBUG=instrument-cra PORT=3001 HTTPS=true DANGEROUSLY_DISABLE_HOST_CHECK=true react-scripts -r @cypress/instrument-cra start",
```

Comment:
is this starting the ui server or starting the cypress stuff?  if it's the later, it should be named something else.  if it's just starting a special version of the ui server this name is fine

Reply:
sounds like the former is the case, ignore the comment